CHANGES_NEXT_RELEASE

```diff
@@ -12,3 +12,4 @@ Fix:  Update PATCH request without payload must return error code 411 (#1257)
 Fix:  Clearer error 'entity not found' and 'conflict too many results' on NGSIv2 update PATCH (#1260)
 Add:  contextBroker standard error log in RPM init script (#1175)
 Fix:  avoid noisy error output in RPM init script (#309)
+Fix:  proxyCoap binary has been removed from RPM (#1202)
```

Comment:
BTW, I have checked that an RPM generated at this branch doesn include the proxyCoap file:

```
[fermin@centollo fiware-orion]$ rpm -qlp /home/fermin/src/fiware-orion/rpm/RPMS/x86_64/contextBroker-0.24.0_next-dev.x86_64.rpm
/etc/cron.d/cron-logrotate-contextBroker-size
/etc/init.d/contextBroker
/etc/logrotate.d/logrotate-contextBroker-daily
/etc/prelink.conf.d/contextBroker.conf
/etc/sysconfig/contextBroker
/etc/sysconfig/logrotate-contextBroker-size
/usr/bin/contextBroker
/usr/share/contextBroker/garbage-collector.py
/usr/share/contextBroker/latest-updates.py
/usr/share/contextBroker/monit_log_processing.py
/usr/share/doc/contextBroker/LICENSE
```